tests/resources/smartsheet/test_funding_sheet2.json

```diff
@@ -0,0 +1,194 @@
+{
+   "accessLevel": "EDITOR_SHARE",
+   "columns": [
+      {
+         "id": 3981351074090884,
+         "index": 0,
+         "title": "Project Name",
+         "type": "TEXT_NUMBER",
+         "validation": false,
+         "version": 0,
+         "width": 371
+      },
+      {
+         "id": 1729551260405636,
+         "index": 1,
+         "primary": true,
+         "title": "Project Code",
+         "type": "TEXT_NUMBER",
+         "validation": false,
+         "version": 0,
+         "width": 150
+      },
+      {
+         "id": 2990791841501060,
+         "index": 2,
+         "options": [
```

Comment:
not all of these are funders - I'm not clear on how you are using this. If it's just for testing, it's fine - but if this permits non-funding institutions to pass into our metadata, I'd prefer to fix it

Reply:
This is just for testing that the abbreviation gets parsed correctly. The funders are defined by the SmartSheet it pulls data from.